packages/utils/src/keystoreKeyVerify.js

```diff
@@ -1,27 +1,22 @@
-import secp256k1 from 'secp256k1';
+const secp256k1 = require('secp256k1');
 
-export default {
-  /**
-   * Verify public key
-   *
-   * @param value
-   * @returns {boolean}
-   */
-  verifyPublicKey(value) {
-    return secp256k1.publicKeyVerify(
-      Buffer.from(value.replace(/^0x/, ''), 'hex'),
-    );
-  },
+/**
+ * Verify public key
+ * @param {string} value
+ * @returns {boolean}
+ */
+const verifyPublicKey = value =>
+  secp256k1.publicKeyVerify(Buffer.from(value.replace(/^0x/, ''), 'hex'));
```

Comment:
In my opinion, previous variant was look more clean and readable. Is this prettier autofix or something manual?
```
const verifyPublicKey = value => secp256k1.publicKeyVerify(
  Buffer.from(value.replace(/^0x/, ''), 'hex')
);

// or
const verifyPublicKey = value => 
  secp256k1.publicKeyVerify(
    Buffer.from(value.replace(/^0x/, ''), 'hex')
  );
```

Reply:
`prettier`